src/fparser/two/Fortran2008.py

```diff
@@ -96,6 +103,279 @@ class Program_Unit(Program_Unit_2003):  # R202
     subclass_names.append("Submodule")
 
 
+class Component_Attr_Spec(Component_Attr_Spec_2003):  # R437
+    '''
+    Fortran 2008 rule R437
+    component-attr-spec is access-spec
+                           or ALLOCATABLE
+                           or CODIMENSION <lbracket> coarray-spec <rbracket>
```

Comment:
Not particularly... `lbracket ... rbracket` is actually `[ ... ]` but I didn't want to use the brackets directly to avoid misinterpretation as optional parts